### ISSUE TITLE
Cannot compile with CUDA 10.2, Kepler

### ISSUE THREAD

**Issue by jzuhone** [gamer-project/gamer#99]
When compiling the latest master (6f4e429f100aed6d9f4447) on NASA/Pleiades with CUDA 10.2 and using KEPLER GPU, I get the following error:

```
Model_Hydro/GPU_Hydro/CUFLU_Shared_FullStepUpdate.cu(185): error: identifier "atomicExch_block" is undefined
```

**Comment by hyschive** [comment]
@jzuhone Does the following replacement for line 185 work?
```
#           if ( __CUDA_ARCH__ >= 600 )
            atomicExch_block( s_FullStepFailure, 1 );
#           else
            atomicExch      ( s_FullStepFailure, 1 );
#           endif
```

**Comment by jzuhone** [comment]
@hyschive it compiles, yes. I will check to see how it runs.

**Comment by hyschive** [comment]
Solved by #100
